.github/workflows/linux_ci.yml

```diff
@@ -115,7 +152,8 @@ jobs:
           git status
           git push
```

Comment:
what does that mean for the following jobs' `actions/checkout@v2`. Do we get the updated version here or the original commit version?

Reply:
I am actually not sure. They should probably get the updated version, but I can't support this claim very well... What I know is that the checkout action checks out the branch, not the commit (I know that because I can commit further, otherwise I'll be in detached head mode).

In any case there should be no difference, at least for now.